app/javascript/packs/application.js

```diff
@@ -1,13 +1,4 @@
-// This file is automatically compiled by Webpack, along with any other files
-// present in this directory. You're encouraged to place your actual application logic in
-// a relevant structure within app/javascript and only use these pack files to reference
-// that code so it'll be compiled.
-
-import Rails from "@rails/ujs"
-import Turbolinks from "turbolinks"
-import * as ActiveStorage from "@rails/activestorage"
-import "channels"
-
-Rails.start()
-Turbolinks.start()
-ActiveStorage.start()
+import "materialize-css/dist/js/materialize"
+import "core-js/stable"
+import "regenerator-runtime/runtime"
+import "../stylesheets/application"
```

Comment:
```suggestion
import "stylesheets/application"
```
воно так підтягне
також розмежуй імпорт локальних файлів і імпорт файлів з ноди новим рядком